pwa/src/templates/templateParts/ratingIndicator/RatingIndicatorTemplate.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import { PieChart } from "react-minimal-pie-chart";
+
+interface RatingIndicatorTemplateProps {
+  component: any;
+}
+
+export const RatingIndicatorTemplate: React.FC<RatingIndicatorTemplateProps> = ({ component }) => {
+  const [rating, setRating] = React.useState<number>(0);
+  const highestPossibleRating = 2;
+
+  React.useEffect(() => {
+    let _rating = 0;
+
+    if (component.legal?.license) {
+      _rating += 1;
+    }
+
+    if (component.maintenance?.contractors) {
+      _rating += 1;
+    }
+
+    setRating(_rating);
+  });
+
+  const calculatePercentage = (rating: number, highestPossibleRating: number) => {
+    const percentage = (100 * rating) / highestPossibleRating;
+    return percentage;
+  };
+
+  const ratingColor = getComputedStyle(document.documentElement).getPropertyValue(
+    "--web-app-color-background-page-header",
+  );
```

Comment:
This is very nicely found, and I don't see another way for us to retrieve our design tokens. However, because we'll be having to extract design-token-values within our javascript more often, I think a service might be a nice addition.

In the `services` folder we could add a new service with (basically doing the exact same thing you've already built here, but making it a bit more generic):

```
export const getTokenValue = (tokenKey: string) => {
  return getComputedStyle(document.documentElement).getPropertyValue(tokenKey);
};
```

Afterwards, we can add a `RatingIndicatorTemplate.modulde.css` with the following export (will be multiple exports, I just did this for one value to test):

```
:export {
  ratingColor: --web-app-color-background-page-header;
}
```

Within our Typescript, we can now retrieve this key (`styles.ratingColor`), and retrieve the value of this key, like so: 

```
labelStyle={{
    fontSize: "25px",
    fontFamily: ratingFont,
    fill: getTokenValue(styles.ratingColor), <== Using our new service and retrieving the key from css
}}
```

I think this will be a more robust (and reusable) way to retrieve the values.